alerts/proxy_drop_executable.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+
+from lib.alerttask import AlertTask
+from query_models import QueryStringMatch, SearchQuery, TermMatch, ExistsMatch
+
+
+class AlertProxyDropExecutable(AlertTask):
+    def main(self):
+
+        superquery = None
+        run = 0
+
+        self.parse_config('proxy_drop_executable.conf', [
+                          'destinationfilter', 'extensions'])
+
+        search_query = SearchQuery(minutes=5)
+
+        search_query.add_must([
+            TermMatch('category', 'squid'),
+            TermMatch('tags', 'squid'),
+            TermMatch('details.proxyaction', 'TCP_DENIED/-')
+        ])
+
+        search_query.add_must_not([
+            QueryStringMatch(
+                'details.destination: /{}/'.format(self.config.destinationfilter)),
+        ])
+
+        # TODO: remove HACK in onAggregation once this works as expected
+        # for extension in self.config.extensions.split(','):
+        #     if run == 0:
+        #         superquery = QueryStringMatch(
+        #             'details.destination: /{0}/'.format(extension))
+        #     else:
+        #         superquery |= QueryStringMatch(
+        #             'details.destination: /{0}/'.format(extension))
+        #     run += 1
+        #
+        # search_query.add_must(superquery)
```

Comment:
@pwnbus please note that there is something in this logic which does not cause an exception but prevents a match.  Initially, I thought it was somehow AND'ing instead of OR'ing, but that does not appear to be the case based on a test where I left only "exe" in the config exceptions.  Something else is happening here.